pkg/token/runtimetokenprovider/provider_server_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build delayedstart
+// +build delayedstart
+
```

Comment:
Not needed. test files should not impact the binary.

Reply:
removed